src/Driver/CockroachDBDriver.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace YlsIdeas\CockroachDb\Driver;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Schema\AbstractSchemaManager;
+use Illuminate\Database\PDO\PostgresDriver;
+use YlsIdeas\CockroachDb\Schema\CockroachSchemaManager;
+
+class CockroachDBDriver extends PostgresDriver
```

Comment:
For consistency this should be `CockroachDb` not `CockroachDB`

```suggestion
class CockroachDbDriver extends PostgresDriver
```

Reply:
Ah yes, you're absolutely right